components/automate-elasticsearch/habitat/plan.sh

```diff
@@ -5,14 +5,19 @@ pkg_name="automate-elasticsearch"
 pkg_description="Wrapper package for core/elasticsearch"
 pkg_origin="chef"
 pkg_version="6.8.3"
-vendor_origin="core"
 pkg_maintainer="Chef Software Inc. <support@chef.io>"
 pkg_license=("Chef-MLSA")
 pkg_upstream_url="https://www.chef.io/automate"
+pkg_source="https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-${pkg_version}.tar.gz"
+pkg_shasum=824078e421c9f7e5ab9c875e4019d9ebfe3ada99db286b54dec090f97d1cbe25
+
 pkg_build_deps=(
-  "${vendor_origin}/elasticsearch/${pkg_version}"
+  core/patchelf
 )
 pkg_deps=(
+  core/busybox-static
```

Comment:
Note this is before coreutils-static which will have an impact on path ordering I think.